open_humans/forms.py

```diff
@@ -207,16 +205,17 @@ class ResetPasswordForm(AllauthResetPasswordForm):
     """
 
     def save(self, request, **kwargs):
+        next_url = request.session.pop('next_url', reverse(settings.LOGIN_REDIRECT_URL))
+
         # Returns email address from cleaned form.
+        # This must be called after retrieveing the redirect url because it
```

Comment:
Maybe this comment is out-of-date, it's unlikely that it's clearing the session.